### DISCUSSION TITLE
Anyway to completely remove a plugin?

### DISCUSSION THREAD

**Discussion by neonvoidx** [LazyVim/LazyVim#1774] [Q&A] [answered]
Not just disable but remove a builtin? i.e i dont want catppuccin, I see no reason that it should keep getting updated if im never going to use it.

**Comment by folke** [comment]
If you disable it, then run `:Lazy` and do clean, it will be gone from your filesystem and won't be installed again

**Reply by neonvoidx** [reply]
```
return {
  { "catppuccin/nvim", enabled = false },
}
```
 then i ran lazy sync, and clean, but cattpuccin doesn't seem disabled, instead it just disabled `nvim`, cattpuccin still shows up under Not Loaded section?
 
![image](https://github.com/LazyVim/LazyVim/assets/25580051/cf85b662-a729-452e-839a-e7f3f4468b20)

**Reply by dpetka2001** [reply, marked as answer]
You should just do `{ "catppuccin", enabled = false },` because of the way it's configured as you can see in the [docs](https://www.lazyvim.org/plugins/colorscheme#catppuccin).

Edit: it's being configured with `name = "catppuccin"`, because otherwise it would be identified by `lazy.nvim` as `nvim` due to the heuristics it uses to identify the module names.

**Reply by neonvoidx** [reply]
ok thanks, sorry for all the inquiries, just recently swapped to vim so still learning

**Reply by dpetka2001** [reply]
No worries. :smile:

**Comment by neonvoidx** [comment]
> You should just do `{ "catppuccin", enabled = false },` because of the way it's configured as you can see in the [docs](https://www.lazyvim.org/plugins/colorscheme#catppuccin).

Ah ok the `name=` takes over completely got it

**Comment by mirsella** [comment]
sorry to bump old thread, but would it be possible to also disable tokyonight ?

**Reply by folke** [reply]
of course, that's exactly the same as for any other plugin

**Reply by dpetka2001** [reply]
You could do 
```lua
  { "LazyVim/LazyVim", opts = { colorscheme = "default" } },
  { "folke/tokyonight.nvim", enabled = false },
```
This disables `tokyonight` and uses the `default` Neovim colorscheme. Feel free to replace `default` with whatever other colorscheme you'd like.

**Reply by mirsella** [reply]
Thanks for the reply ! I tried multiple things, and it seems like I have a problem with the colorscheme option.
i have like the docs suggest
```lua
return {
  {
    "LazyVim/LazyVim",
    opts = {
      -- loading another colorscheme in another place
      colorscheme = function() end,
    },
    -- // lazyvim errors if disabled
    -- { "folke/tokyonight.nvim", enabled = false },
    { "catppuccin", enabled = false },
  },
}
```
but the `colorscheme` option seems to not do anything (still loading tokyonight even with any other string, my colorscheme, or this `function() end`.

should i open a issue ?

**Reply by dpetka2001** [reply]
I just tried the code snippet I mentioned in my previous comment and it correctly disabled `tokyonight` and loads the Neovim `default` colorscheme. Did you try the code snippet as is? It should load the blackish Neovim default colorscheme.

**Reply by mirsella** [reply]
oh frick i missed the intendation of the table, i just had to move a line for it to work with the code i had. thanks have a great day !
(if you look at the code the catppuccin and tokyonight line are inside the lazyvim config)
